APE_LTS.rst

```diff
@@ -0,0 +1,109 @@
+Ending Long Term Support Releases
+---------------------------------
+
+author: Thomas Robitaille
+
+date-created: 2023 February 16 <replace with the date you submit the APE>
+
+date-last-revised: 2023 February 16 <keep this up to date anytime something changes>
+
+date-accepted: <replace with accepted date>
+
+type: Process
+
+status: Discussion
+
+
+Abstract
+--------
+
+This APE proposes to no longer designate any releases of the core package as being
+'long term support' (LTS) and provide guidance on how to handle API changes and deprecations.
+
+
+Detailed description
+--------------------
+
+`APE 2: Astropy Release Cycle and Version Numbering <https://doi.org/10.5281/zenodo.1043887>`_ describes the release cycle
+for the core astropy package, which consists of major releases every 6 months that
+add new features (e.g., v5.2), and releases in between this that are limited to fixing bugs
+or updating documentation (e.g., v5.2.1). Every two years, major releases are designated
+as being 'long term support' (e.g., v5.0) and we have committed to releasing bugfix releases
+for this version (e.g., v5.0.3) for up to two years.
+
+At the start of the project, having long term support releases was critical for acceptance by
+large organizations/projects for which stability is paramount, as the core package went through
+significant API changes from version to version due to the youth of the project and long term
+support releases were a way to shield against these changes, or at least reduce disruption to
+be every two years.
+
+LTS releases are however not free:
+
+* While we use infrastructure to automate backporting fixes to release branches,
+  in practice the LTS branch diverges more and more from the main development
+  branch and automated backports are more likely to fail as time goes on,
+  requiring human effort to manually do the backports, which can sometimes be
+  non-trivial.
+
+* Even once fixes are backported to the LTS branch, carrying out releases is not
+  fully automated and requires manual steps, so having to do
+  bugfix releases for both the latest release branch and LTS branch puts an extra
+  burden on the core package release team.
+
+In addition, it is not clear that LTS releases are needed anymore - now that the
+core package is over ten years old, it has reached a much higher level of
+stability. No other main package in the scientific Python ecosystem has long
+term support releases, and there is no longer a strong argument for having this
+in astropy either.
+
+This APE proposes the following:
+
+* No longer designate any releases as LTS
+* Keep the 6-month release cycle and bump releases to x.0 every two years
+* Refer to the x.0 releases every two years (e.g. v6.0) as 'major'
+  releases, other releases every 6 months (e.g. v6.1) as 'minor', and bug fixing
+  releases (e.g. v6.1.2) as 'bugfix releases'
+* Minimize as much as possible any changes that break API or change results in
```

Comment:
This requires us to re-educate all the maintainers with write access to the repo. We wouldn't want inconsistency in how this is decided based on who happened to merge the PRs. Should this APE also lay out how these new rules should be followed?

Reply:
Agree - see my latest comment in the main discussion. I think we'll need to add a section about this.